pkgs/build-support/gcc-wrapper/setup-hook-stdinc.sh

```diff
@@ -0,0 +1,38 @@
+export NIX_CC=@out@
```

Comment:
@ttuegel This is a quite good explanation for uncommented code :smile: I think it would be great if you put it in the file